dask/tests/test_delayed.py

```diff
@@ -40,47 +40,47 @@ def __dask_postcompute__(self):
         return tuple, ()
 
 
+@pytest.mark.filterwarnings("ignore:The dask.delayed:UserWarning")
 def test_to_task_dask():
-    with warnings.catch_warnings(record=True):
```

Comment:
Subtle interaction between pytest and the warnings module, `catch_warnings` just prevents the warnings from being displayed. They still are emitted, and so pytest will still fail with `-Werror`. We would need to either use `warnings.catch_warnings()` with a `warnings.simplefilter`, or `pytest.mark.filterwarnings`.

Reply:
Good to know.  Thanks for the public service announcement.  I'll endeavor to remember this for the future (but I suspect that I'll probably forget too occasionally)